addons/mail/static/src/components/thread_viewer/thread_viewer_tests.js

```diff
@@ -81,8 +81,11 @@ QUnit.test('dragover files on thread with composer', async function (assert) {
         name: "General",
         public: 'public',
     });
-    const threadViewer = this.env.models['mail.thread_viewer'].create({ thread: [['link', thread]] });
-    await this.createThreadViewerComponent(threadViewer, { hasComposer: true });
+    const threadViewerContainer = this.env.models['mail.thread_viewer_container'].create({
+        hasThreadViewer: true,
+        thread: [['link', thread]],
+    });
+    await this.createThreadViewerComponent(threadViewerContainer.threadViewer, { hasComposer: true });
```

Comment:
With suggestion to rename models to `mail.thread_view` and `mail.thread_viewer`, the `ThreadViewer` component should be renamed to `ThreadView`.

Would make a bit more sense why you need a `mail.thread_viewer` to use a `ThreadView` component, because you can't have a view without a viewer!

Reply:
Ok but will do that as prep PR tomorrow